OpenRA.Mods.Common/Traits/World/EditorResourceLayer.cs

```diff
@@ -113,6 +115,17 @@ public void UpdateCell(CPos cell)
 				Dirty.Add(cell + d);
 		}
 
+		void CalculateNetWorth()
```

Comment:
Ah, *selfslap* `CVec.Directions` does not contain 0,0. No wonder this doesn't work.